packages/fuels/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fuels",
-  "version": "0.43.1",
+  "version": "0.43.0",
```

Comment:
Should not have been adjusted, will sort before live PR

Reply:
What do you mean? It's `0.43.1` on `master` ([here](https://github.com/FuelLabs/fuels-ts/blob/c8698f5f8d48ea8680f02719190334550616d6a3/packages/fuels/package.json#LL3C3-L3C23)).